libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -1332,6 +1332,10 @@ void ContractCompiler::appendModifierOrFunctionCode()
 
 	if (codeBlock)
 	{
+		bool previousChecked = m_context.checkedArithmetics();
+		// TODO test that checks are also applied for initializing state variables
```

Comment:
Can you actually use `unchecked` for state variable initialization?

Reply:
no